src/DefinitionResolver.php

```diff
@@ -234,6 +235,18 @@ public function createDefinitionFromNode(Node $node, string $fqn = null): Defini
             $def->documentation = $this->getDocumentationFromNode($node);
         }
 
+        $def->parameters = [];
+        if (property_exists($node, 'parameters') && $node->parameters) {
```

Comment:
I will need @roblourens help for this - I have no idea what to check for. From what I gather it is a trait.